transport/http/handler_test.go

```diff
@@ -135,7 +135,7 @@ func TestHandlerInternalFailure(t *testing.T) {
 
 	rpcHandler := transporttest.NewMockHandler(mockCtrl)
 	rpcHandler.EXPECT().Handle(
-		gomock.Any(),
+		transporttest.NewContextMatcher(t, time.Second),
```

Comment:
The context matcher will test that the Context this function is called with has a TTL of one second give or take a few milliseconds.